Makefile

```diff
@@ -0,0 +1,206 @@
+.PHONY: help
```

Comment:
I am generally not a fan of Makefiles like this.
Its not adding anything over `go generate ./...` and `go test`.

We do have a `docker-bake.hcl` which can be useful for executing linters and certain tests and is generally very helpful for lazily building the frontend and feeding that into a build of a spec file without having to save the frontend to an image.